awscli/examples/networkflowmonitor/get-query-status-workload-insights-top-contributors-data.rst

```diff
@@ -0,0 +1,13 @@
+**To retrieve the status of the query**
+
+The following ``get-query-status-workload-insights-top-contributors-data`` example displays the current status of the query in the specified account. ::
+
+    aws networkflowmonitor get-query-status-workload-insights-top-contributors-data \
+        --scope-id e21cda79-30a0-4c12-9299-d8629d76d8cf \
+        --query-id 4333754d-8ae1-3f29-b6b7-c36db2e7f8ac
+
+Output::
+
+    {
+        "status": "SUCCEEDED"
+    }
```

Comment:
Every example needs a for more information link.
https://aws.github.io/aws-cli/docs_styleguide.html#for-more-information-link